skypy/power_spectrum/_camb.py

```diff
@@ -72,29 +76,26 @@ def camb(wavenumber, redshift, cosmology, A_s, n_s):
                        omch2=cosmology.Odm0 * h2,
                        omk=cosmology.Ok0,
                        TCMB=cosmology.Tcmb0.value,
-                       mnu=np.sum(cosmology.m_nu.value),
+                       mnu=np.sum(cosmology.m_nu.to_value(u.eV)),
                        standard_neutrino_neff=cosmology.Neff
                        )
 
-    # camb requires redshifts to be in decreasing order
-    redshift_order = np.argsort(redshift)[::-1]
+    # camb interpolator requires redshifts to be in increasing order
+    redshift_order = np.argsort(redshift)
```

Comment:
In complete opposition to the rubbish I was saying before...
```suggestion
    # camb interpolator requires redshifts and wavenumber to be in increasing order
    redshift_order = np.argsort(redshift)
    wavenumber_order = np.argsort(wavenumber)
```
... and then below the requisite change as well.